db/migrate/20170919211745_create_subnets.rb

```diff
@@ -0,0 +1,11 @@
+class CreateSubnets < ActiveRecord::Migration[5.0]
+  def change
+    create_table :subnets do |t|
+      t.string :ems_ref
+      t.string :name
+      t.integer :lan_id
+    end
+
+    add_foreign_key :subnets, :lans
```

Comment:
we don't use foreign keys, so please remove

Reply:
We should use foreign keys though, for 5.0, I will be adding them to multiple places. Since it's a new table, it might be worth to treat the DB the right way. ;-) (subnet should always be under network, it's one of the easy models)

That being said, it might open an ugly can of worms, so might be good to have a full release to solve it. :-)

Reply:
Sure, once we go to 5.0, we can discuss...the original reason was because of replication which is a moot point, but I don't feel comfortable adding them until a major release